<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogGeneralConfiguration</class>
 <widget class="QDialog" name="DialogGeneralConfiguration">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General Configuration</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelProgramOnline">
       <property name="text">
        <string>Program Online</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonProgramOnline">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Offline</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelResetCounters">
       <property name="text">
        <string>Reset Counters</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButtonResetCounters">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelTriggerPin">
       <property name="text">
        <string>Trigger Pin</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBoxTriggerPin"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelOnlinePin">
       <property name="text">
        <string>OnlinePin</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBoxOnlinePin"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelBusyPin">
       <property name="text">
        <string>Busy Pin</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBoxBusyPin"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelOkPin">
       <property name="text">
        <string>OK Pin</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="comboBoxOkPin"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelNOKPin">
       <property name="text">
        <string>NOK Pin</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="comboBoxNOKPin"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="buttonOk">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
